### PR TITLE
Add capability to update package.json from projen in create-release action

### PR DESCRIPTION
**Pull request by quesabe**
Closes PLA-186.

### CODE REVIEW

File: create-release/action.yml [hunk 1]
```diff
@@ -59,11 +62,21 @@ runs:
...
       shell: bash
       run: npm version ${{ steps.bump-version.outputs.new_tag }}
+    - name: Update main package.json via projen
+      if: inputs.update-root-package_json == 'true' && inputs.projen == 'true'
+      shell: bash
+      run: |
+        fileName=$([ -f .projenrc.ts ] && echo ".projenrc.ts" || echo ".projenrc.js")
+        regex="package.addField('version', '[0-9]\.[0-9]\.[0-9]')"
+        newVersion=$(echo ${{ steps.bump-version.outputs.new_tag }} | cut -c 2-6)
+        sed -i -e "s/$regex/package.addField('version', '$newVersion')/" $fileName
```
Comment (gvidon) [inline create-release/action.yml:78]: @quesabe just thought — can we maybe do it like this in `.projenrc.ts`:
```typescript
project.addField('version', process.env.PACKAGE_VERSION || <read from package.json>)
```

What do you think about such approach?

**Review by gvidon — CHANGES_REQUESTED** [review]

File: create-release/action.yml (hunk 1 above)
Comment (quesabe) [reply create-release/action.yml:78]: Sounds promising - that gonna simplify the action code a lot. Let me try it and see if there is any downside to this.

**Comment by gvidon** [comment]
@quesabe I guess based on [this](https://github.com/ottofeller/templates/pull/180) the PR is no longer relevant? If so please feel free closing it.

**Comment by quesabe** [comment]
> @quesabe I guess based on [this](https://github.com/ottofeller/templates/pull/180) the PR is no longer relevant? If so please feel free closing it.

Sure. I'd just like to hold it until we are confident that the other solution works. I'll keep an eye on this one and will close it once we have the task complete.

**Pull request closed without merging**